src/modules/mavlink/mavlink_main.cpp

```diff
@@ -1926,6 +1932,10 @@ Mavlink::task_main(int argc, char *argv[])
 	_mode = MAVLINK_MODE_NORMAL;
 	bool _force_flow_control = false;
 
+#ifdef __PX4_POSIX_BBBLUE
+	_mavlink_wifi_name = __PX4_BBBLUE_DEFAULT_MAVLINK_WIFI;
```

Comment:
Can we make this more generic and remove the ifdefs and avoid the board-specific init?
`_mavlink_wifi_name` -> `_network_interface_name`

Reply:
Made it generic.